tests/large-alignment-test.c

```diff
@@ -0,0 +1,18 @@
+/*
+   Tests relocation and data in a large (>1MB) struct
+*/
+
+#include "gtest_like_c.h"
+
+float __attribute__((aligned (32))) large_aligned_array[3];
+
+int large_alignment()
+{
+  large_aligned_array[0] = 1;
+  return 1;
```

Comment:
I am missing some sort of check for the alignment?

Reply:
The test validates that code with aligning which could be compiled and array is accessible. The array size is the same, allocated space by compiler is different. It might be checked by using binutils tools. For now just a simple test to validate the changes.

Reply:
I am thinking of something like this [https://stackoverflow.com/questions/1898153/how-to-determine-if-memory-is-aligned](https://stackoverflow.com/questions/1898153/how-to-determine-if-memory-is-aligned), but OK.

Reply:
The best option will be using dump for the disassembler and analyzing assembly code.

Reply:
I think that would be an overkill.